app/Http/Controllers/Assets/AssetsController.php

```diff
@@ -149,8 +149,11 @@ public function store(ImageUploadRequest $request) : RedirectResponse
 
             if (! empty($settings->audit_interval)) {
                 $asset->next_audit_date = Carbon::now()->addMonths($settings->audit_interval)->toDateString();
-            }
-
+            } 
+	    else {
```

Comment:
We'd probably want to check for `filled()` here, so we don't null it out if it's left blank.